### ISSUE TITLE
Platypus: Missing main landmark

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#2068]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page is missing a main landmark.

---

https://dequeuniversity.com/rules/axe/4.3/landmark-one-main?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Blind, Deafblind, Mobility
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A
